game-core/src/main/java/games/strategy/triplea/delegate/data/MustMoveWithDetails.java

```diff
@@ -29,6 +30,6 @@ public MustMoveWithDetails(final Map<Unit, Collection<Unit>> mapping) {
   }
 
   public Collection<Unit> getMustMoveWithForUnit(final Unit unit) {
-    return mapping.getOrDefault(unit, List.of());
```

Comment:
Hmm think it would be better if we just put en empty list there instead of null in the first place, but we're just passing the map somewhere else, so yeah...